kubernetes-examples/multitenant/base/kroxylicious-config.yaml

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kroxylicious-config
+data:
+  config.yaml: |
+    filters:
+    - type: MultiTenant
+    adminHttp:
+      endpoints:
+        prometheus: {}
+    virtualClusters:
+      devenv1:
+        targetCluster:
+          bootstrap_servers: my-cluster-kafka-bootstrap:9092
+        clusterNetworkAddressConfigProvider:
+          type: PortPerBroker
+          config:
+            bootstrapAddress: minikube:30192
+        logNetwork: false
+        logFrames: false
+      devenv2:
```

Comment:
Should we call this cluster `marketing` as that matches up with the comments in the deployment.